notebooks/Section1_3-Model_Comparison.ipynb

```diff
@@ -30,7 +30,7 @@
     "import matplotlib.pyplot as plt\n",
```

Comment:
<p>Didn't even know this changed, probably on arviz side. But maybe best to have the most future-proof code in here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fonnesbeck/Bayes_Computing_Course/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Not sure what you are referring to. The warning?</p>

Reply:
<p>Yeah, exactly.</p>